### PR TITLE
added new icons

### PR DESCRIPTION
**Pull request by subeep**
new icons pack for the application to have vector graphics.
from issue [#139 ](https://github.com/Crazy-Marvin/ToDont/issues/139).
And updated the launcher foreground and background svg files.

### CODE REVIEW

**Comment by subeep** [comment]
@CrazyMarvin ohk done.

File: app/src/debug/res/mipmap-anydpi-v26/ic_launcher.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+    <background android:drawable="@drawable/ic_launcher_background"/>
+    <foreground android:drawable="@drawable/ic_launcher_foreground"/>
```
Comment (CrazyMarvin) [inline app/src/debug/res/mipmap-anydpi-v26/ic_launcher.xml:4]: May you please add the monochrome icon too which was introduced with Android 13?
`<monochrome android:drawable="@drawable/ic_launcher_monochrome"/>`

**Comment by CrazyMarvin** [comment]
Thank you so much, @subeep!
It would be great if you could add support for Themed Icons on Android 13+ (see my comment). 
Other than that it should be fine to merge. Any objections, @arafaatqureshi?

File: app/src/debug/res/mipmap-anydpi-v26/ic_launcher.xml (hunk 1 above)
Comment (subeep) [reply app/src/debug/res/mipmap-anydpi-v26/ic_launcher.xml:4]: Sure, I will commit that in a next pull.

**Pull request merged**